packages/augur-simplified/src/modules/common/top-nav.tsx

```diff
@@ -133,7 +145,7 @@ export const SettingsButton = () => {
                         }
                       }
                     }}
-                    placeholder={slippage}
+                    placeholder="custom"
                     max="100"
```

Comment:
max slippage is 1000%, need to update value in custom input. can be done in later PR or this one.